src/QMCWaveFunctions/TrialWaveFunction.cpp

```diff
@@ -38,14 +38,15 @@ typedef enum
 
 static const std::vector<std::string> suffixes{"V", "VGL", "accept", "NLratio", "recompute", "buffer", "derivs"};
 
-TrialWaveFunction::TrialWaveFunction(const std::string& aname)
+TrialWaveFunction::TrialWaveFunction(const std::string& aname, bool tasking)
```

Comment:
const?

Reply:
const matters more for reference type.
Here is just a scalar and passed in by value and local to the function, I don't really feel the need to add const.
We have many integer arguments in the code, I don't feel the need to update them by adding this type of const.